docs/infrastructure-macos.md

```diff
@@ -39,16 +39,38 @@ On the Mac, go to System Settings -> Network and disable WiFi.
 Then, in the network settings, go to Ethernet -> Details -> DNS, and click "+" under "DNS Servers" to add the IPv4 addresses of alimesos01, alimesos02 and alimesos03 as DNS servers.
 Remove other DNS servers (including the automatic CERN central ones).
 
+You will have to register them in LanDB too, the network will redirect you to the right page automatically.
+
+### Set up the Mac's hostname
+
+Make sure the Mac knows itself by it's real hostname, as it can have issues
+with DNS resolution otherwise. You can do this by going to System Preferences
+-> General -> About, and changing the computer name to the short hostname of
+the Mac (e.g. `alibuildmac09`).
+
+### Enable SSH and VNC
+
+To start both the SSH and VNC servers, enable remote login in System Preferences -> General -> Sharing -> Remote Login, and Screen Sharing.
+
+Additionally, you may want to add the proper SSH keys to the `alibuild` user's `authorized_keys` file.
+
 ### Prevent the Mac from going to sleep
 
 By default, Macs will go into a low-power state after a while without interactive use, which interrupts the CI build process.
 To prevent this, change the following settings in System Settings:
 
-In Energy Saver, enable both "Start up automatically after a power failure" and "Wake for network access".
+In Energy, enable "Start up automatically after a power failure", "Wake for network access", and "Prevent computer from sleeping automatically when the display is off".
```

Comment:
This is fine, we should find the command line equivalent of all those (something like `defaults write ...`).